samples/smoketest/Program.py

```diff
@@ -1,13 +1,8 @@
-from key_vault_secrets import KeyVault
-from storage_blob import StorageBlob
-from event_hubs import EventHub
-from cosmos_db import CosmosDB
+import smoke_test
 
-print("==========================================")
-print("      AZURE TRACK 2 SDKs SMOKE TEST")
-print("==========================================")
-
-KeyVault().Run()
-StorageBlob().Run()
-EventHub().Run()
-CosmosDB().Run()
+try:
+    import smoke_test_async
+except SyntaxError:
```

Comment:
First line of that module is `import asyncio`, so I expect on 2.7 you'd see `ImportError`.

Reply:
Tested, it's a SyntaxError